AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -330,17 +350,26 @@ private extension UPIComponent {
             continueButton.title = localizedString(.QRCodeGenerateQRCode, configuration.localizationParameters)
         }
         
-        updateButtonState()
-    }
-    
-    func updateButtonState() {
-        continueButton.enabled = canSubmit()
+        hideError()
     }
     
     func focusVpaInput() {
         vpaInputItem.focus()
     }
     
+    private func showError() {
+        errorItem.isHidden.wrappedValue = false
+        UIAccessibility.post(
+            notification: .announcement,
+            argument: "\(localizedString(.errorTitle, configuration.localizationParameters)): \(errorItem.message ?? "")"
+        )
+    }
+
+    private func hideError() {
+        errorItem.message = nil
+        errorItem.isHidden.wrappedValue = true
+    }
+    
     func canSubmit() -> Bool {
```

Comment:
Many functions seem to be missing `internal` keyword or github messing up?

Reply:
We're in a private extension here - removed the other keywords